src/ui/FileViewer/__tests__/utils.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { mapFileViewerComponentProps } from '../utils';
+import { FileViewerComponentProps, ViewerTypes } from '../types';
+import { FILE_INFO_LIST } from '../data.mock';
+
+const noop = () => { /* noop */ };
+
+describe('Global-utils/getBorderRadiusForImageRenderer', () => {
```

Comment:
Describe -> `mapFileViewerComponentProps ` instead of `getBorderRadiusForImageRenderer`